swig/perl/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 include(UseSWIG)
 
 set(CMAKE_SWIG_FLAGS "-module" "openscap_pm")
+set_property(SOURCE openscapPERL5_wrap.c PROPERTY COMPILE_OPTIONS "-w")
```

Comment:
will this work on other compilers with different command line arguments? is `-w` universal?

Reply:
I don't know. Could we reduce that only to GCC/Clang ?

Reply:
I guess we could. It would only mean that if you compile the SWIG bindings using different compilers than GCC/Clang the warnings will be displayed.